docs/hub/security-two-fa.md

```diff
@@ -35,6 +35,10 @@ In Hugging Face Hub:
 
 If you entered the correct pin, the Hub displays a list of recovery codes. Download them and keep them in a safe place.
 
+<Tip>
+After configuring 2FA, your account enters a 30-day check-up period. To exit this period, successfully complete 2FA within those 30 days. If you do not complete 2FA during this time, you will be prompted to perform 2FA in an existing session on the 30th day.
```

Comment:
A bit convoluted IMO, maybe we can do something simpler

Would this be a correct assertion?
```suggestion
You will be prompted for 2FA on every new session, and every 30 days for active sessions.
```

Reply:
Yes it is

Reply:
"session" might not be the clearest wording here

Reply:
```suggestion
You will be prompted for 2FA every time you log in, and after that every 30 days.
```

Reply:
or cc @Michellehbn x)